integration/kubernetes/init.sh

```diff
@@ -40,6 +40,31 @@ wait_pods_ready()
 	done
 }
 
+build_custom_stress_image()
+{
+	info "Build custom stress image"
+	registry_image="docker.io/library/registry:2"
+	arch=$("${SCRIPT_PATH}/../../.ci/kata-arch.sh")
+	if [[ "${arch}" == "ppc64le" || "${arch}" == "s390x" ]]; then
+		# that image is not built for these architectures
+		registry_image="docker.io/ibmcom/registry:2.6.2.5"
```

Comment:
We try to avoid hard-coding version numbers like this, preferring to add them to one of the two versions databases:

- https://github.com/kata-containers/kata-containers/blob/main/versions.yaml
- https://github.com/kata-containers/tests/blob/main/versions.yaml

You can then use the [`get_version()`](https://github.com/kata-containers/tests/blob/main/.ci/lib.sh#L148) function to query the version. For example, https://github.com/kata-containers/tests/blob/main/.ci/install_rust.sh#L19.